.github/workflows/test.sh

```diff
@@ -63,10 +65,17 @@ else
     test_args+=(--suite testdata_seek-stress)
 fi
 
-if [[ -n $frame_delay ]]; then
+if [[ -n $frame_delay && -n $negative_stride ]]; then
+    # Frame delay is tested with two threads; negative strides with one.
+    echo "Error: frame_delay and negative_stride options can't be used together. Exiting."
```

Comment:
Can frame delays not be used with negative strides?  Or we're just testing it this one way?

Reply:
I don't know enough about how these features interact so so the safest and simplest solution I could come up with was to support the test flows that dav1d has and make sure they don't interfere negatively. We can always come back and do something more sophisticated when there is a clear need.